### DISCUSSION TITLE
Problemas no som quando liga o emulador do android

### DISCUSSION THREAD

**Discussion by lcsteil** [Flutterando/forum#163] [Perguntas e Respostas]
Olá!

Estou tendo problemas no som quando ligo o emulador do Android.

O som do computador simplesmente desaparece ou fica um ruido bem alto.

Quando desligo o emulador, o som volta ao normal.

Estou utilizando Linux popOS

Alguém já passou por esse problema?

**Comment by ffeliciodeveloper** [comment]
Bom dia!
Tudo tranquilo?!

Eu tive esse problema na minha máquina e isso envolve a memória da máquina.
O emulador consome muito dos recursos de memória.
Um ssd ajuda, mas resolve poucos os problemas.

A empresa forneceu uma máquina com 16gb de ram, mas em dados momentos a máquina chora se eu abro vários emuladores.

**Comment by euclidesgc** [comment]
Tenta isso aqui que resolveu pra mim:

https://stackoverflow.com/questions/67000474/when-i-start-android-emulator-the-audio-on-my-mac-desktop-stops

> I found easier way than accepted answer from @Ivo Stoyanov to solve this. Just open the emulator config file (for my mac it's ~/.android/avd/{emulator_name}/config.ini and set

**Comment by renanzdm** [comment]
Para mim toda vez que abro o emulador preciso rodar esse comando `systemctl --user restart pipewire.service`
